_posts/blog/2017-04-10-paris-traceroute-brownout.md

```diff
@@ -0,0 +1,89 @@
+
+<div class="table-responsive" markdown="1">
+
+| year  |  week  |  ndt_tests  |  pt_tests  |   %   |
+| :-----|:------:|:-----------:|:----------:|:------|
+| 2016  | 21     | 391624      | 220025     | 56.18 |
+| 2016  | 22     | 406367      | 271845     | 66.90 |
+| 2016  | 23     | 416730      | 275954     | 66.22 |
+| 2016  | 24     | 412697      | 323775     | 78.45 |
+| 2016  | 25     | 423601      | 358566     | 84.65 |
+| 2016  | 26     | 402461      | 321197     | 79.81 |
+| 2016  | 27     | 454424      | 336444     | 74.04 |
+| 2016  | 28     | 468149      | 318792     | 68.10 |
+| 2016  | 29     | 529151      | 291702     | 55.13 |
+| 2016  | 30     | 654513      | 309584     | 47.30 |
+| 2016  | 31     | 884118      | 372715     | 42.16 |
+| 2016  | 32     | 968719      | 377798     | 39.00 |
+| 2016  | 33     | 992234      | 388339     | 39.14 |
+| 2016  | 34     | 1042148     | 382655     | 36.72 |
+| 2016  | 35     | 1086401     | 363672     | 33.47 |
+| 2016  | 36     | 1125381     | 337876     | 30.02 |
+| 2016  | 37     | 1116448     | 286531     | 25.66 |
+| 2016  | 38     | 1259814     | 197173     | 15.65 |
+| 2016  | 39     | 1310179     | 93825      | 7.16  |
+| 2016  | 40     | 1329701     | 76366      | 5.74  |
+| 2016  | 41     | 1293060     | 70755      | 5.47  |
+| 2016  | 42     | 1243503     | 59420      | 4.78  |
+| 2016  | 43     | 1176804     | 56772      | 4.82  |
+| 2016  | 44     | 1026661     | 47130      | 4.59  |
+| 2016  | 45     | 1181540     | 34044      | 2.88  |
+| 2016  | 46     | 1205690     | 46432      | 3.85  |
+| 2016  | 47     | 1303431     | 86086      | 6.60  |
+| 2016  | 48     | 1329139     | 92356      | 6.95  |
+| 2016  | 49     | 1449058     | 95055      | 6.56  |
+| 2016  | 50     | 1583852     | 121391     | 7.66  |
+| 2016  | 51     | 1654478     | 112102     | 6.78  |
+| 2016  | 52     | 1578605     | 80632      | 5.11  |
+| 2016  | 53     | 1742942     | 110150     | 6.32  |
+| 2017  | 1      | 1811016     | 88584      | 4.89  |
+| 2017  | 2      | 1844633     | 92833      | 5.03  |
+| 2017  | 3      | 1772941     | 118736     | 6.70  |
+| 2017  | 4      | 1762644     | 160980     | 9.13  |
+| 2017  | 5      | 1790433     | 168677     | 9.42  |
+| 2017  | 6      | 1808985     | 600434     | 33.19 |
+| 2017  | 7      | 1788070     | 1550172    | 86.70 |
+| 2017  | 8      | 1858179     | 1565195    | 84.23 |
+| 2017  | 9      | 1862311     | 1663829    | 89.34 |
+| 2017  | 10     | 1851940     | 1747511    | 94.36 |
+| 2017  | 11     | 1969324     | 1864045    | 94.65 |
+| 2017  | 12     | 1898877     | 1754154    | 92.38 |
+
+</div>
+
+![NDT tests and Paris Traceroute tests per week, June 2016 - Mar 2017]({{ site.baseurl }}/images/blog/pt-brownout-graph.png)
```

Comment:
I'm unclear about what you're suggesting. The image is included in the post, where else were you thinking it should go?